rearrange/environment.py

```diff
@@ -700,6 +723,11 @@ def drop_held_object(self) -> bool:
             action_space=self.action_space,
             action_fn=self.drop_held_object,
             thor_action="DropHandObject",
+            default_thor_kwargs={
+                "autoSimulation": False,
+                "randomMagnitude": 0.0,
```

Comment:
What is this?

Reply:
By default, `"DropHandObject"` adds some random noise to the angular momentum of the dropped object. `randomMagnitude=0.0` turns this off. `autoSimulation=False` just makes it so we control the physics updates when dropping an object.